lib/scripts/test_poste_deliveries.rb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+require 'cdo/db'
+require 'cdo/poste'
+
+# Create a [poste_messages.id -> template] hash of poste_messages so we don't
+# need to query for each delivery
+templates = POSTE_DB[:poste_messages].to_hash(:id, :name).map do |id, name|
+  # copied from Deliverer.load_template
+  path = Poste.resolve_template(name)
+  raise ArgumentError, "#{name.inspect} template wasn't found." unless path
+  template = Poste::Template.new path
+  [id, template]
+end.to_h
+
+# for each email we've delivered in the last year, simply attempt to render the
+# email so we will trigger the existing "ActionView/TextRender incompatibility"
+# warnings for that email.
+#
+# We do this because many of our emails are seasonal, so if we want to test
+# this thoroughly with the existing passive approach, we'd have to wait like a
+# year to be sure.
+POSTE_DB[:poste_deliveries].where("sent_at > '2019-01-01'").each do |delivery|
```

Comment:
Thinking about whether it'd be a-bad-thing :tm: if someone accidentally ran this script in the future.  Possible it's a good idea to dynamically set this to a year ago, instead of having a hard-coded date which will increase the scope of this script over time.

Reply:
my plan to mitigate that danger was to just remove this script after I'm done